frame/contracts/src/migration.rs

```diff
@@ -15,459 +15,542 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::{BalanceOf, CodeHash, Config, Pallet, TrieId, Weight};
-use codec::{Decode, Encode};
+// TODO gate
```

Comment:
Needs to be resolved before merge.

Reply:
actually do you know how to gate this modules and make them public for try-runtime only
you can use block {} here so you need 
one	#[cfg(feature = "runtime-benchmarks")] and #[cfg(not(feature = "runtime-benchmarks"))] for each. Do you write a macro for that or is there something easier?